CFC_WebApp/main/googlemaps.py

```diff
@@ -0,0 +1,493 @@
+#!/usr/bin/env python
+
+# Copyright 2009 John Kleint
+#
+# This is free software, licensed under the Lesser Affero General 
+# Public License, available in the accompanying LICENSE.txt file.
+
+
+"""
+An easy-to-use Python wrapper for the Google Maps and Local Search APIs.
+
+* **Geocoding**: convert a postal address to latitude and longitude
```

Comment:
Cool. You might want to add a note to that effect at the top.

Have you considered either submitting a pull request so that the original one is up to date as well?